DEVELOPMENT.md

```diff
@@ -107,23 +107,28 @@ Use [Codeception](https://codeception.com/) for running end-2-end tests in the b
    - In shell, run `chromedriver --version`. _Note: If you are using OS X, it may prevent this program from opening. Open "Security & Privacy" and allow chromedriver_.
    - Run `chromedriver --version` again. _Note: On OS X, you may be prompted for a final time, click "Open"_. When you can see the version, chromedriver is ready.
 
-### 2. Headless Site Setup
+### 2. Front-end Setup
 
-1. From within the headless site `examples/getting-started` copy `.env.test.sample` to `.env.test`.
-   - If you are using the provided Docker build, you will not need to adjust any variables in the `.env.testing` file; else, you can adjust the environment variables as needed.
+1. Create the following `.env.test.local` in `examples/next/getting-started`.
```

Comment:
Does it need to be `.env.test.local` or just `.env.test`?

Reply:
Either one works since we're using [dotenv-flow](https://www.npmjs.com/package/dotenv-flow). They generally recommend committing `.env.test` and not committing `.env.test.local`, which is why I picked the `.local` version. However, we already exclude all `.env*` files from git rather than just the `.local` ones, so I'd be fine with the shorter `.env.test`. I'll make that update.